kotlin/configure-subnetwork-trace/README.md

```diff
@@ -20,13 +20,13 @@ Example barrier conditions for the default dataset:
 
 1. Populate the traversability scope list with the enum values from `UtilityTraversabilityScope`.
 2. Create and load a `UtilityNetwork` with a feature service URL, then get an asset type and a tier by their names.
-3. Populate the choice list for the comparison source with the non-system defined `Definition.networkAttributes`.  Populate the choice list for the comparison operator with the enum values from `UtilityAttributeComparisonOperator`.
+3. Populate the choice list for the comparison source with the non-system defined `UtilityNetworkDefinition.networkAttributes`.  Populate the choice list for the comparison operator with the enum values from `UtilityAttributeComparisonOperator`.
 4. Create a `UtilityElement` from this asset type to use as the starting location for the trace.
-5. Update the selected barrier expression and the checked options in the UI using this tier's `TraceConfiguration`.
-6. When 'Network Attribute' is selected, check whether it has a `CodedValueDomain`, and if so, populate the choice list for the comparison value with its `CodedValues`.  Otherwise, display a free-form textbox for entering an attribute value.
-7. When 'Add' is clicked, create a new `UtilityNetworkAttributeComparison` using the selected comparison source, operator, and selected or typed value. Use the selected source's `UtilityNetworkAttribute.DataType` to convert the comparison value to the correct data type.
-8. If `Traversability.barriers` is not empty, create a `UtilityTraceOrCondition` with the existing barriers and the new comparison from step 8.
-9. When 'Trace' is clicked, create `UtilityTraceParameters` passing in `UtilityTraceType.SUBNETWORK` and the default starting location.  Set its `UtilityTraceConfiguration` with the modified options, selections, and expression; then run a `UtilityNetwork.traceAsync(...)`.
+5. Update the selected barrier expression and the checked options in the UI using this tier's `UtilityTraceConfiguration`.
+6. When 'Network Attribute' is selected, if its `Domain` is a `CodedValueDomain`, populate the choice list for the comparison value with its `CodedValues`.  Otherwise, display a free-form textbox for entering an attribute value.
```

Comment:
whoops, looks like `Domain` snuck back in again. Remember this isn't part of our API, I think you had corrected this in an earlier commit. Like this:

> When 'Network Attribute' is selected, check whether it has a `CodedValueDomain`, and if so...